src/components/Block/ResultsError.vue

```diff
@@ -0,0 +1,15 @@
+<script setup>
+import { useI18n } from 'vue-i18n';
+const { t } = useI18n();
+</script>
+
+<template>
+  <Block :title="$t('results')" icon="warning" iconClass="text-red">
```

Comment:
Was thinking "Results error" maybe for the title?

Reply:
The icon has a pointer cursor on hover but isn't linking to anything

Reply:
I remember now that was actually this sparkle icon for the old plugin settings modal. So yea no linking needed anymore, at least for now. Removed the submit event and a tag.

Reply:
Regarding the title.... thought about that too. But only for a split second and then... naaaaahhhh it's ok.